apps/desktop/src/lib/backend/ipc.ts

```diff
@@ -64,3 +71,22 @@ export function listen<T>(event: EventName, handle: EventCallback<T>) {
 	const unlisten = listenTauri(event, handle);
 	return async () => await unlisten.then((unlistenFn) => unlistenFn());
 }
+
+export class Tauri {
+	listen<T>(event: EventName, handle: EventCallback<T>) {
+		const unlisten = listenTauri(event, handle);
+		return async () => await unlisten.then((unlistenFn) => unlistenFn());
+	}
+
+	async checkUpdate(): Promise<UpdateResult> {
+		return await checkUpdate();
+	}
+
+	async onUpdaterEvent(callback: (status: UpdateStatusResult) => void): Promise<UnlistenFn> {
+		return await onUpdaterEvent(callback);
+	}
+
+	async currentVersion(): Promise<string> {
+		return await getVersion();
+	}
+}
```

Comment:
It seems difficult to mock these functions without having an object, I tried several ways with no luck.

Will also move this into its own class and just reference the `ipc.ts` methods instead of duplicating them. Eventually we should move all services over to using an object that can be mocked for testing.